ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/SVI_PROFILE_TESTS.yml

```diff
@@ -0,0 +1,363 @@
+          113:
+            name: svi_profile_tests_113_description
+            enabled: true
+            ip_address_virtual: 10.1.13.1/24
+            structured_config:
+              description: set from svi.structured_config
+            profile: struct_config_child_profile_3
+
+          # Expected results:
+          # - description: set from structured_config on svi_profile.structured_config
+          114:
+            name: svi_profile_tests_114_description
+            enabled: true
+            ip_address_virtual: 10.1.14.1/24
+            profile: struct_config_child_profile_3
+
+          # Expected results:
+          # - description: set from structured_config on svi_parent_profile.structured_config
+          115:
+            name: svi_profile_tests_115_description
+            enabled: true
+            ip_address_virtual: 10.1.15.1/24
+            profile: struct_config_child_profile_4
+
+#### ---- Test enheritance of svi igmp_snooping_enabled ----- ###
+# Tests logic for network services: ip igmp snooping
+# Expected result for all tests is that igmp snooping is enabled for all VLANs
+
+          # igmp_snooping_enabled set on svi to true, child_profile and parent_profile set to false
+          210:
+            name: igmp_snooping_enabled_210
+            enabled: true
+            ip_address_virtual: 10.2.10.1/24
+            igmp_snooping_enabled: true
+            profile: igmp_child_profile_1
+
+          # igmp_snooping_enabled set on child_profile to true and parent_profile set to false
+          211:
+            name: igmp_snooping_enabled_211
+            enabled: true
+            ip_address_virtual: 10.2.11.1/24
+            profile: igmp_child_profile_2
+
+          # igmp_snooping_enabled set on parent_profile to true
+          212:
+            name: igmp_snooping_enabled_212
+            enabled: true
+            ip_address_virtual: 10.2.12.1/24
+            profile: igmp_child_profile_3
+
+#### ---- Test enheritance of svi ip_virtual_router_addresses ----- ###
+# Tests logic for network services: static-routes.
+# Expected result for all tests is vxlan disabled for all VLANs
+
+          # vxlan set on svi to false, child_profile and parent_profile set to true
+          310:
+            name: vxlan_disabled_310
+            enabled: true
+            ip_address_virtual: 10.3.10.1/24
+            vxlan: false
+            profile: vxlan_child_profile_1
+
+          # vxlan set on child_profile to false and parent_profile set to true
+          311:
+            name: vxlan_disabled_311
+            enabled: true
+            ip_address_virtual: 10.3.11.1/24
+            profile: vxlan_child_profile_2
+
+          # vxlan set on parent_profile to false
+          312:
+            name: vxlan_disabled_312
+            enabled: true
+            ip_address_virtual: 10.3.12.1/24
+            profile: vxlan_child_profile_3
+
+#### ---- Test enheritance of svi ip_virtual_router_addresses ----- ###
```

Comment:
```suggestion
#### ---- Test inheritance of svi ip_virtual_router_addresses ----- ###
```

Reply:
is it the same test as before??? looks like this one is router and the previous one should be about vxlan